framework/web_app/src/models/StockLevel.ts

```diff
@@ -1,4 +1,5 @@
 export type StockLevel = {
     description: string;
```

Comment:
Lmk if you want this resolved

Reply:
was tossing up over whether or not to do this, but in the end figured because we're using IDs which are strings we could very easily run into "this doesn't work and i don't know why" because ID != name

an easier approach would be to have the models nested on each other in one to one relationships, but i'm kind of curious to see how far we can get with just IDs

for the foreseeable future i don't think we'd implement user defined values for this, just can't see the need for it

Reply:
also fantastic ideas, this project would be a mess without you :stuck_out_tongue: